staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -220,6 +220,7 @@ func (g *gcePersistentDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume
 				},
 			},
 			AccessModes: []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce},
+			VolumeMode:  &fsMode,
```

Comment:
This is potentially unnecessary given [this workaround](https://github.com/kubernetes/kubernetes/commit/c9fc26d85ef755f7b9c15f039e469857d75f5aec#diff-94e527d26427ceb4f990f05f76952656). Without the workaround, inline volumes used to fail earlier in the call to GetVolumeMode. Either [1] simply remove this and re-test or [2] see if the setting of PersistentVolumeFilesystem for CSI translation can be made more generic across all plugins - not just GCE PD.

Reply:
Implemented [2] above in https://github.com/kubernetes/kubernetes/pull/83098/files#diff-3b2a81c2c4c59f6e82eded08cd00af5cR70-R83